GVFS/GVFS.Common/NuGetUpgrade/NuGetFeed.cs

```diff
@@ -20,7 +20,8 @@ namespace GVFS.Common.NuGetUpgrade
     public class NuGetFeed : IDisposable
     {
         // This is the SHA256 Certificate Thumbrint we expect packages from Microsoft to be signed with
-        private const string TrustedMicrosoftCertFingerprint = "3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE";
+        private const string TrustedMicrosoftCertFingerprintOld = "3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE";
+        private const string TrustedMicrosoftCertFingerprintNew = "AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27";

```

Comment:
_If_ we were going to start expanding this list in the long-term, then I would recommend using `...Fingerprint_2020_10` and `...Fingerprint_2020_12` or something. (Yes, the old one has been a round a long time, I'm just thinking that October 2020 is the last time we successfully used it.)

However, this is probably not our long-term strategy so `Old`/`New` is fine.